frontend/integration-tests/protractor.conf.ts

```diff
@@ -90,7 +90,8 @@ const testSuites = {
     'tests/monitoring.scenario.ts',
     'tests/alertmanager.scenario.ts',
     'tests/crd-extensions.scenario.ts',
-    'tests/devconsole/pipeline.scenario.ts',
+    '../packages/dev-console/integration-tests/tests/dev-perspective.scenario.ts',
+    '../packages/dev-console/integration-tests/tests/pipeline.scenario.ts',
```

Comment:
This shouldn't be necessary. The plugin can add its tests to e2e in package.json. Here's an example:

https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/package.json#L19-L21

Reply:
Will this allow the above tests to run on every pr

Reply:
Yes, if you add to the e2e suite in this way, the tests will run against PRs and gate merge.

Reply:
Acknowledged, I'll edit my pr.